examples/player/simid_player.html

```diff
@@ -42,6 +42,24 @@
       simidPlayer.playAd();
     }
 
+    function closeAd() {
+      if (!simidPlayer) {
+        initAd();
+      }
+      simidPlayer.closeAd();
+    }
+
+    function skipAd() {
+      if (!simidPlayer) {
+        initAd();
+      }
+      simidPlayer.skipAd();
+    }
+
+    function fatalError() {
+      simidPlayer.fatalError();
```

Comment:
Why doesn't this one have a guard? What if simidPlayer doesn't exist when this is clicked?

Reply:
fixed